src/assets/Azure.Core.Shared/RequestContentHelper.cs

```diff
@@ -87,5 +87,26 @@ public static RequestContent FromDictionary(IDictionary<string, BinaryData> dict
 
             return content;
         }
+        /*
+        public static RequestContent FromString(string value)
+        {
+            var content = new Utf8JsonRequestContent();
+            content.JsonWriter.WriteStringValue(value);
+            return content;
+        }
+
+        public static RequestContent FromBool(bool value)
+        {
+            var content = new Utf8JsonRequestContent();
+            content.JsonWriter.WriteBooleanValue(value);
+            return content;
+        }
+        */
+        public static RequestContent FromObject(object value)
+        {
+            var content = new Utf8JsonRequestContent();
+            content.JsonWriter.WriteObjectValue(value);
+            return content;
+        }
```

Comment:
Could we just use `RequestContent.Create` for such cases like primitive types like strings and booleans?
I believe this could work just fine (but we need to validate)
```
var b = true;
var r = RequestContent.Create(b); // this should creates a request content: true (without quotes)
```
I wonder how we should behave when we get a string.
If the content of the string is "foo", there is no doubt that the content should be "foo" (with quotes).
What if the content of the string is a json string like "{}", should we have the quotes in RequestContent?

Reply:
If the content of the string is "foo", `RequestContent.Create` will return `foo` (without quotes).

Reply:
OK then we have to have this method I believe.